### ISSUE TITLE
Issues playing without CUDA

### ISSUE THREAD

**Issue by mmilk90** [CorentinJ/Real-Time-Voice-Cloning#54]
Excuse my inexperience, but I tried to use your software since it looked very appealing to me. 
I use an AMD GPU, so I followed the steps on installing torch without it. However, I keep getting this error message.

![CUDA error](https://user-images.githubusercontent.com/53124383/61582236-ba3c5d00-aaed-11e9-9a9e-e702e04db66e.PNG)

It seems my only failure is that it does not playback.
Is it even possible to use it without CUDA? What can I do to circumvent this error?

**Comment by CorentinJ** [comment]
It may be possible, but you would have to change calls to cuda() to get that to work. I don't know how torch handles other GPUs. You would also need to change the device the weights are stored on.

**Comment by daydayupcheng** [comment]
I run this tool successfully on my local machine without GPU. You need to replace all the codes which include cuda.

**Comment by GeeKh04** [comment]
> I run this tool successfully on my local machine without GPU. You need to replace all the codes which include cuda.

please can you explain how can i do that ?

**Comment by slavaGanzin** [comment]
> I run this tool successfully on my local machine without GPU. You need to replace all the codes which include cuda.

If it works on CPU, why not making this optional?
I saw this in other projects like (pseudocode):

```python
if torch.cuda and not options.force_cpu:
  torch.device = torch.cuda
else:
  torch.device = torch
```

and then rewrite all torch.cuda to torch.device. I think it's look a bit like a hack, but it works and it's easy to refactor

**Comment by AteetKumar** [comment]
> I run this tool successfully on my local machine without GPU. You need to replace all the codes which include cuda.

How did you replace all the cuda codes? Can you please explain ? That would be great help.

Thanks and regards,
Ateet Kumar

**Comment by stinct9** [comment]
For anyone wanting to run this without CUDA. It's possible by removing the Cuda codes. I have been successfully able to run it without CUDA. Will try to raise a PR if I get some time.

**Comment by blue-fish** [comment]
@mmilk90 I wanted to let you know we have implemented CPU support in #366 so you do not need CUDA to run the voice cloning toolbox. So, exactly what you wanted! The setup process is a little bit easier now too. Please pull the latest and try again.
